elevate-cpanel

```diff
@@ -3516,6 +3504,13 @@ EOS
         return;
     }
 
+    sub _gather_php_usage ($self) {
+        my $php_get_vhost_versions = Elevate::EA4::php_get_vhost_versions();
+        Elevate::StageFile::remove_from_stage_file('php_get_vhost_versions');
+        Elevate::StageFile::update_stage_file( { php_get_vhost_versions => $php_get_vhost_versions } );
```

Comment:
do we want to store it if `undef`?

Reply:
it shouldn't hurt anything to store it if it is `undef` since the post_leapp code that restores it returns unless it gets an aref back from this